test-clients/scripts/updateNodeTransfer.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# script to be called by platform regression scrip to transfer files to update Node
+#
+#
+set -eE
+UPDATE_NODE_IP_ADDRESS=""
+NODE0_IP_ADDRESS=""
+CONFIG_FILE_PATH=""
+copyFilesToUpdateNode() {
+  # copy config.txt , File0.0.150 and saved state to update nodes to start from the state
+  set -x
+  sudo scp -o StrictHostKeyChecking=no -i services-regression.pem -p $CONFIG_FILE_PATH ubuntu@$UPDATE_NODE_IP_ADDRESS:/home/ubuntu/remoteExperiment
```

Comment:
I think without 'sudo', this command should work too.

Reply:
Agreed. removed it. Thanks @ljianghedera